### PR TITLE
add java and python solutions to cses-1081

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/cses-1081.mdx [hunk 1]
```diff
@@ -44,6 +46,55 @@ int main() {
 	cout << ans << endl;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+```java
+import java.util.Scanner;
+
+public class Main {
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:55]: could you rename this to some form of the problem name

File: solutions/cses-1081.mdx [hunk 2]
```diff
@@ -44,6 +46,55 @@ int main() {
...
+	public static int gcd(int a, int b) {
+		return b == 0 ? a : gcd(b, a % b);
+	}
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:61]: use bufferedreader

File: solutions/cses-1081.mdx [hunk 3]
```diff
@@ -96,6 +149,81 @@ int main() {
...
+import java.util.Scanner;
+
+public class Main {
+	public static final int MAX_VAL = 1000000;
+
+	// divisors[i] = stores the count of numbers that have i as a divisor
+	public static int divisors[] = new int[MAX_VAL + 1];
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:168]: yeah

File: solutions/cses-1081.mdx [hunk 4]
```diff
@@ -148,21 +276,61 @@ int main() {
...
 ```
 </CPPSection>
+
+<JavaSection>
+```java
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+	public static final int MAXX = 1000005;
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:286]: is the +5 necessary

File: solutions/cses-1081.mdx [hunk 5]
```diff
@@ -148,21 +276,61 @@ int main() {
...
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:289]: .

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/cses-1081.mdx (hunk 1 above)
Comment (876pol) [reply solutions/cses-1081.mdx:55]: fixed

File: solutions/cses-1081.mdx (hunk 2 above)
Comment (876pol) [reply solutions/cses-1081.mdx:61]: oops forgot that existed 💀
bufferedreader does a lot better, but three cases still tle -- couldnt seem to figure out a way to get it to ac

**Comment by bqi343** [comment]
this file was moved to https://github.com/cpinitiative/usaco-guide/blob/master/solutions/gold/cses-1081.mdx in master btw

File: solutions/cses-1081.mdx [hunk 6]
```diff
@@ -96,6 +147,83 @@ int main() {
...
+```java
+import java.io.*;
+import java.util.*;
+
+public class CommonDivisors {
+
+	public static final int MAX_VAL = 1000000;
+
+	// divisors[i] = stores the count of numbers that have i as a divisor
+	public static int divisors[] = new int[MAX_VAL + 1];
```
Comment (SansPapyrus683) [inline solutions/cses-1081.mdx:165]: does this have to be static

**Comment by SansPapyrus683** [comment]
ayo what

**Pull request closed without merging**